yaksh/templates/yaksh/course_modules.html

```diff
@@ -17,6 +17,7 @@
         <center>{{ msg }}</center>
     </div>
 {% endif %}
+<b>Grade: {% if grade %} {{ grade }} {% else %} Will be available once the course is complete {% endif %}</b>
```

Comment:
`Will be available once the course is complete` is not the proper message. since my course is completed but just because my obtained grade does not fit into available grading system this message shows up.

Reply:
For now it will pick the default grading system!
So grade will always be seen.
Need to add a way by which we can deactivate the grading system, will do with the above mentioned issue.